docs/source/json-api/index.rst

```diff
@@ -103,6 +103,8 @@ From a DAML project directory:
             prefix -- URL prefix,
             directory -- local directory that will be mapped to the URL prefix.
             Example: "prefix=static,directory=./static-content"
+      --leak-passwords-firesheep-style
+            DEV MODE ONLY (not recommended for production). Allow connections without a reverse proxy providing HTTPS.
```

Comment:
I'm not sure if the reference to firesheep is appropriate. What do you think about a more neutral name like `--allow-tokens-without-tls`? I wouldn't be surprised if this breaks `create-daml-app` can you please test that it still works after this change, for instance by using `daml-sdk-head` and doing the first few steps of the getting started guide.

Reply:
The current name comes from @garyverhaegen-da , how's this alternative?

Reply:
I also do not think the firesheep reference is necessarily appropriate. I'm also slowly realizing there are a lot of people in the internet that may be too young to understand it 😢

I do think it needs to be "scary", though. I like cURL's `--insecure`, but could see something more specific, like `--allow-insecure-tokens` or the like.

Reply:
> I'm also slowly realizing there are a lot of people in the internet that may be too young to understand it

I seem to fall into that group :slightly_smiling_face:

Reply:
TIL I've been listening to Security Now for over 10 years.

[Security Now, Episode #272: Firesheep](https://twit.tv/shows/security-now/episodes/272).

Reply:
Changed to `--allow-insecure-tokens` in abb4034c5e6835fa794b209061e178da8b3954dd.